docs/source/user_guide/from_to_dbms.rst

```diff
@@ -0,0 +1,107 @@
+====================
+From/to other DBMSes
+====================
+.. currentmodule:: databricks.koalas
+
+
+The APIs interacting with other DBMSes in Koalas is slightly different from the ones in pandas
```

Comment:
nit: are?

The APIs interacting with other DBMSes in Koalas **are** slightly ...